lib/utils/cli.go

```diff
@@ -522,3 +523,21 @@ type PredicateError struct {
 func (p PredicateError) Error() string {
 	return fmt.Sprintf("%s\nCheck syntax at https://goteleport.com/docs/setup/reference/predicate-language/#resource-filtering", p.Err.Error())
 }
+
+func FormatAlertOutput(alert types.ClusterAlert) string {
+	var buf bytes.Buffer
+	if runtime.GOOS == constants.WindowsOS {
+		// TODO(timothyb89): Due to complications with globally enabling +
+		// properly resetting Windows terminal ANSI processing, for now we just
+		// disable color output. Otherwise, raw ANSI escapes will be visible to
+		// users.
+		fmt.Fprint(&buf, alert.Spec.Message)
+	} else {
+		if alert.Spec.Severity == types.AlertSeverity_HIGH {
+			fmt.Fprint(&buf, Color(Red, alert.Spec.Message))
+		} else {
+			fmt.Fprint(&buf, Color(yellow, alert.Spec.Message))
```

Comment:
Is it normal that `Red` is capitalized here and `yellow` is not?

Reply:
Both Yellow and yellow are defined, swapped this to the uppercase one for consistency.